### PR TITLE
Third round of PDF edits

### PR DESCRIPTION
**Pull request by rudeboybert**

### CODE REVIEW

**Comment by rudeboybert** [comment]
Let me know your thoughts on bacffe4291fc31c944fff70085615553f7dc6a1f. I'm proposing indicating "captured vs not" using alpha and blue vs orange. This did necessitate changing to `theme_light()` for these three HTML output visualizations, or else there isn't enough contrast.

**Comment by rudeboybert** [comment]
@ismayc When I ask for a review, in particular let me know your thoughts on bacffe4291fc31c944fff70085615553f7dc6a1f. I'm proposing indicating "captured vs not" using alpha and blue vs orange. This did necessitate changing to `theme_light()` for these three HTML output visualizations, or else there isn't enough contrast.

File: 08-confidence-intervals.Rmd [hunk 1]
```diff
@@ -1511,7 +1530,9 @@ sample_of_cis <- percentile_cis_by_n %>%
   mutate(sample_row = 1:10)
 
 ggplot(sample_of_cis) +
-  geom_point(aes(x = point_estimate, y = sample_row)) +
+  # Doesn't make sense to show point_estimate center for percentile confidence
```
Comment (ismayc) [inline 08-confidence-intervals.Rmd]: @rudeboybert Good catch! Copy-and-paste error on my part.

File: 09-hypothesis-testing.Rmd [hunk 2]
```diff
@@ -1244,27 +1249,38 @@ In the denominator $s_a$ and $s_r$ are the *sample standard deviations* of the a
...
+  ggplot(aes(x = domain, y = value, color = Distribution)) +
+  geom_line() +
+  scale_color_viridis(discrete = TRUE) +
+  theme_light() +
+  theme(
+    axis.title.y = element_blank(),
+    axis.title.x = element_blank(),
+    axis.text.y = element_blank(),
+    axis.ticks.y = element_blank()
+  )
```
Comment (ismayc) [inline 09-hypothesis-testing.Rmd]: Nice!

**Comment by ismayc** [comment]
@rudeboybert Let's merge these in and start a new PR (from `pdf-edits-round-3` to `master` if you are OK with this. I'd like to make sure to keep PRs small and not have long standing reviews if possible.

**Pull request merged**